config_templates/gretel/transform/default.yml

```diff
@@ -16,18 +21,11 @@ models:
                   - person_name
                   - credit_card_number
                   - phone_number
-                  - us_social_security_number
                   - email_address
-                  - custom/*
+                    location
               transforms:
                 - type: fake
                 - type: redact_with_char
                   attrs:
                     char: X
-label_predictors:
-  namespace: custom
-  regex:
-    user_id:
-      patterns:
-        - score: high
-          regex: "user_[\\d]{5}"
```

Comment:
Thank you for doing that! I just remembered we had this random regex in our default, which didn't make much sense.